### ISSUE TITLE
Weird loading issue

### ISSUE THREAD

**Issue by Todi13** [ZaneYork/SMAPI-Android-Installer#512]
I really want to play Star dew valley expanded but when I add the shop tile framework mod to the mods folder and try to load or create a new game the game just freezes and returns back to the home screen of my tablet(exits the game), but the game works perfectly when I remove the shop tile framework mod. I really want to play Star dew valley expanded and this mod(shop tile framework) is needed for that. Could someone please help me?? PLS.

**Comment by Zencee** [comment]
shop tile framework works well in mine maybe its due to device specs.

**Comment by Caeljin** [comment]
The Producer Framework Mod version 1.4.1 work well here

**Comment by Caeljin** [comment]
Shop Tile Framework 1.0.9 work well here

**Comment by Todi13** [comment]
Then I wonder what the problem is, is it space core ??? I had a problem with that too but I  solved it. Let me check my android version and please tell me what kind of device specs could make the mod not to work.

**Comment by Caeljin** [comment]
The loading time with spacecore increase and still have the bug in the museum with the collection
You can upload your log and post here
https://smapi.io/log

**Comment by Todi13** [comment]
Oh well I will have to wait some more. Thanks slot everyone and I hope I didn't waste your time.

**Comment by Caeljin** [comment]
Yes (removing the S.V.E) and the spacecore work but with bugs (increase loading times and museum reset)

**Comment by Todi13** [comment]
@Caeljin So I can only wait for better versions of smapi right??, or is there a solution to this problem??? Please tell me if there is

**Comment by Todi13** [comment]
Also advice to everyone, while waiting for the next version of smapi, you guys can play this game called TERRARIA. It's really nice and I would recommend it for fans of stardew valley. That's if you guys don't know the game already.

**Comment by Caeljin** [comment]
Spacecore still need be fixed for SVE work properly and without issues :/

**Comment by Todi13** [comment]
Okay then
